### PR TITLE
Re-enable integration tests for elb_network_lb

### PR DESCRIPTION
**Pull request by tremble**
Depends-On: https://github.com/ansible-collections/amazon.aws/pull/940

##### SUMMARY

- Re-enables integration tests for elb_network_lb
- Moves from hard-coded SSL certs to generating them on the fly
- Fixes bug where `ip_address_type` in return value wasn't updated

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

tests/integration/targets/elb_network_lb

### CODE REVIEW

File: plugins/modules/elb_network_lb.py [hunk 1]
```diff
@@ -405,7 +404,10 @@ def create_or_update_elb(elb_obj):
     # ip address type
     snaked_elb['ip_address_type'] = elb_obj.get_elb_ip_address_type()
 
-    elb_obj.module.exit_json(changed=elb_obj.changed, **snaked_elb)
+    elb_obj.module.exit_json(
+        changed=elb_obj.changed,
+        load_balancer=snaked_elb,
```
Comment (jatorcasso) [inline plugins/modules/elb_network_lb.py:409]: Not too familiar with this module but is this `load_balancer` return supposed to be here? I couldnt find it in the DOCS section

Comment (tremble) [reply plugins/modules/elb_network_lb.py:409]: I forgot to update the docs :)

I was planning on nesting it as per https://github.com/ansible-collections/amazon.aws/pull/865 :)

**Review by alinabuzachis — APPROVED** [review]

File: tests/integration/targets/elb_network_lb/aliases [hunk 2]
```diff
@@ -1,6 +1,3 @@
 # reason: missing-policy
```
Comment (tremble) [inline tests/integration/targets/elb_network_lb/aliases:1]:
```suggestion
```

File: tests/integration/targets/elb_network_lb/aliases [hunk 3]
```diff
@@ -1,6 +1,3 @@
...
-# reason: broken
-# The SSL cert stored in the test has expired.  Should be dynamically generated.
-disabled
```
Comment (tremble) [inline tests/integration/targets/elb_network_lb/aliases:2]:
```suggestion
```

**Pull request merged**